src/styles/variables.scss

```diff
@@ -15,6 +15,7 @@ $z-index-modal: $z-index-header + 2;
 $z-index-loader: $z-index-modal - 1; // Modals should always be above the loader
 
 $bg-blur-radius: 4px;
+$card-border-radius: 16px;
```

Comment:
On line 5 and 6 we have variables related to border radiuses with different naming pattern.